docs/library/pyb.DAC.rst

```diff
@@ -122,3 +122,12 @@ Methods
      dac2 = DAC(2)
      dac1.write_timed(buf1, pyb.Timer(6, freq=100), mode=DAC.CIRCULAR)
      dac2.write_timed(buf2, pyb.Timer(7, freq=200), mode=DAC.CIRCULAR)
+
+Constants
+---------
+
+.. data:: DAC.NORMAL
+          DAC.CIRCULAR
+
+   A tuple of (subclass, protocol, max packet length, polling interval, report descriptor) to set appropriate values for a USB
```

Comment:
This sentence is out of place here.

Reply:
sorry C&P error, I have added the correct descriptions,